meta/tags.json

```diff
@@ -5,42 +5,51 @@
     "longname": "High Impact",
     "recommend_ui": true,
     "description": "Polls that the Governance Facilitators have judged high-impact",
-    "related_link": "https://forum.makerdao.com/t/introducing-impact-estimations/14267",
+    "related_link": "https://forum.sky.money/t/introducing-impact-estimations/14267",
     "precedence": 1
   },
   {
     "id": "risk-parameter",
     "shortname": "Parameter",
     "longname": "Risk Parameter",
     "recommend_ui": true,
-    "description": "Polls that change risk parameters within the Maker Protocol",
+    "description": "Polls that change risk parameters within the Sky Protocol",
     "precedence": 2
   },
   {
     "id": "technical",
     "shortname": "Tech",
     "longname": "Technical",
     "recommend_ui": true,
-    "description": "Polls relating to technical changes to the Maker Protocol",
+    "description": "Polls relating to technical changes to the Sky Protocol",
     "precedence": 3
   },
   {
-    "id": "mips",
-    "shortname": "MIP",
-    "longname": "MIP",
+    "id": "weekly",
+    "shortname": "Weekly",
+    "longname": "Weekly Atlas Edit",
     "recommend_ui": true,
-    "description": "Polls related to the Maker Improvement Proposal Process defined in MIP0",
-    "related_link": "https://mips.makerdao.com/mips/details/MIP0",
+    "description": "Polls relating to Weekly Atlas Edit Proposals",
+    "related_link": "https://sky-atlas.powerhouse.io/A.1.10.2_Atlas_Edit_Weekly_Cycle/4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59|0db30308",
     "precedence": 4
   },
+  {
+    "id": "monthly",
+    "shortname": "Monthly",
+    "longname": "Monthly Atlas Edit",
+    "recommend_ui": true,
+    "description": "Polls relating to Monthly Atlas Edit Proposals",
```

Comment:
Same here. It'd be more accurate to use "Monthly Cycle Atlas Edit" or "Atlas Edit Monthly Cycle Proposal".